### PR TITLE
[ZEPPELIN-3878] Browser IE 11  not loading Note

### PR DESCRIPTION
**Pull request by vinayakshedgeri**
### What is this PR for?
Previous version of the zeppelin supports IE 11 when  we updated to zeppelin 0.8.0, Zeppelin web was not loading properly saying the error promise undefined .

### What type of PR is it?
[Bug Fix ]

### Todos
* [ ] - 

### What is the Jira issue?
* https://issues.apache.org/jira/projects/ZEPPELIN/issues/ZEPPELIN-3878

### How should this be tested?
click on any of the existing note in IE 11 browser. 
Observation  : Note not loading or not reponsive
you can check the error in the console
* First time? Setup Travis CI as described on https://zeppelin.apache.org/contribution/contributions.html#continuous-integration
* Strongly recommended: add automated unit tests for any new or changed behavior
* Outline any manual steps to test the PR here.

### Questions:
* Does the licenses files need update? No
* Is there breaking changes for older versions? Yes
* Does this needs documentation? No

### CODE REVIEW

**Comment by jongyoul** [comment]
Thank you. BTW, could you please enable your travis-ci for testing Zeppelin? :-)

**Comment by jongyoul** [comment]
FYI, https://travis-ci.org/vinayakshedgeri

File: zeppelin-web/karma.conf.js [hunk 1]
```diff
@@ -38,7 +38,7 @@ module.exports = function(config) {
     // list of files / patterns to load in the browser
     files: [
       // for polyfill
-      'node_modules/babel-polyfill/dist/polyfill.js',
+      //'node_modules/babel-polyfill/dist/polyfill.js',
```
Comment (felixcheung) [inline zeppelin-web/karma.conf.js]: is this going to break ployfill if commented out?

File: zeppelin-web/src/app/app.js [hunk 2]
```diff
@@ -15,6 +15,8 @@
  * limitations under the License.
  */
 
+import "babel-polyfill";
```
Comment (jongyoul) [inline zeppelin-web/src/app/app.js]: AFAIK, this will cover the commented one.

**Comment by jongyoul** [comment]
BTW, I've got the exact same issue like you when I build branch-0.8. Please wait a min to investigate this issue.

**Comment by vinayakshedgeri** [comment]
> BTW, I've got the exact same issue like you when I build branch-0.8. Please wait a min to investigate this issue.
@jongyoul  any update one this

**Comment by jongyoul** [comment]
Unfortunately, not. I don't have much time to check this issue. Can you build it on your local machine by cleaning all of maven local repositories like `rm -rf ~/.m2`?

**Comment by jongyoul** [comment]
Please rebase your commit. I fixed it now

**Comment by Reamer** [comment]
Please us Microsoft Edge.

**Pull request closed without merging**